CommandLine/testing/GitHub-ImageDiff.sh

```diff
@@ -21,29 +21,58 @@ function imgur_upload {
               --form "image=@$1")
 }
 
-imgur_response=$( imgur_upload './enigma_draw_test.png' )
-imgur_master_response=$( imgur_upload '/tmp/enigma-master/enigma_draw_test.png' )
-imgur_diff_response=$( imgur_upload '/tmp/enigma_draw_diff.png' )
-
-imgur_url=$(echo $imgur_response | jq --raw-output '.data."link"' )
-imgur_master_url=$(echo $imgur_master_response | jq --raw-output '.data."link"' )
-imgur_diff_url=$(echo $imgur_diff_response | jq --raw-output '.data."link"' )
-
-echo $imgur_url
-echo $imgur_master_url
-echo $imgur_diff_url
-
-gh_comment="Regression tests have indicated that graphical changes have been introduced. \
+gh_comment_header="Regression tests have indicated that graphical changes have been introduced. \
 Carefully review the following image comparison for anomalies and adjust the changeset accordingly.\n\
 \n\
 $TRAVIS_PULL_REQUEST_SHA | Master | Diff\n\
---- | --- | ---\n\
-<a href='$imgur_url'><img alt='Image Diff' src='$imgur_url' width='200'/></a>|\
-<a href='$imgur_master_url'><img alt='Image Diff' src='$imgur_master_url' width='200'/></a>|\
-<a href='$imgur_diff_url'><img alt='Screen Save' src='$imgur_diff_url' width='200'/></a>\n"
-
-curl -u $bot_user':'$bot_password \
-  --header "Content-Type: application/json" \
-  --request POST \
-  --data '{"body":"'"$gh_comment"'"}' \
-  $pullrequest
+--- | --- | ---\n"
+
+gh_comment_images=""
+
+images=$(ls ./test-harness-out/*.png)
```

Comment:
This iterates the files in the PR rather than at base (master), right?  I believe both directories should have the same contents; it would probably be good to verify that. Otherwise, iterate the files from base, instead, so that if files were deleted, they will be detected as 100% diff, and if files were added (new tests), they will not cause error. You can just pipe the two `ls` results to `cmp` or whatever, using named pipes.